rust/cubesql/cubesql/egraph-debug-template/src/index.tsx

```diff
@@ -18,6 +18,7 @@ import type {
     NodeProps,
 } from 'reactflow';
 import 'reactflow/dist/style.css';
+import { z } from 'zod';
```

Comment:
Probably, not a big deal but still: 
We already have a [joi](joi.dev/) scheme validator in our codebase, so maybe we can reuse it here too? I know, `node_modules` is a black hole and could hold any number of packages) but... If we can reduce and reuse...Why not?

Reply:
I'd rather switch to `zod` in those places :sweat_smile: 

Joi does not introduce static types in TypeScript along with building parser, so one need to manually support and ensure compatibility for both. There are solutions like `joi-to-typescript`, but, given the usual process of working with codegen in JS, it's not very comfortable.

Also egraph debugger is a completely separate package from monorepo, so it wouldn't pollute anything with unnecessary dependencies in "usual" case, and it would install them second time even if dependencies were same.